.github/workflows/docker_publish.yml

```diff
@@ -0,0 +1,44 @@
+
+on:
+  push:
+    branches: ['gh_action_gh_container_registry']  # todo replace with main once we know this is working
+
+env:
+  REGISTRY: 
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    strategy:
+      matrix:
+       name : [
+          pymoab-py2-18.04,
+          pymoab-py3-18.04,
+          pymoab-visit-py2-18.04,
```

Comment:
GH is trying to do these in parallel, but the visit build depends on the non visit build.  When it fails (for reasons I don't understand) the other jobs are stopped.
1. we could try this without the visit version for now
2. can we make the visit version depend on the others?

Reply:
I've tried to include an ``if success`` option in the yml